### PR TITLE
Fix BIA-13, load server from variable

### PR DESCRIPTION
**Pull request by korczis**
Signed-off-by: Tomas Korcak <korczis@gmail.com>

### CODE REVIEW

**Comment by korczis** [comment]
@cvengros Can you review this, please?

File: lib/gooddata/rest/connection.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ module GoodData
   module Rest
     # Wrapper of low-level HTTP/REST client/library
     class Connection
-      DEFAULT_URL = 'https://secure.gooddata.com'
+      DEFAULT_URL = ENV['GD_GEM_SERVER'] || 'https://secure.gooddata.com'
```
Comment (cvengros) [inline lib/gooddata/rest/connection.rb]: `GD_GEM_SERVER` is a bit weird name - I first though that this is some GoodData's rubygems server. Maybe just `GD_SERVER` or `GD_URL`?

**Comment by cvengros** [comment]
when you incorporate the comment, will be ok to merge

**Comment by korczis** [comment]
@cvengros Updated

**Comment by cvengros** [comment]
ok to merge

**Pull request merged**